sklearn/_loss/tests/test_loss.py

```diff
@@ -1091,3 +1126,50 @@ def test_loss_pickle(loss):
     assert loss(y_true=y_true, raw_prediction=raw_prediction) == approx(
         unpickled_loss(y_true=y_true, raw_prediction=raw_prediction)
     )
+
+
+@pytest.mark.parametrize("p", [0, 1, 1.5, 2, 3])
+def test_tweedie_log_identity_consistency(p):
+    half_tweedie_log = HalfTweedieLoss(power=p)
+    half_tweedie_identity = HalfTweedieLossIdentity(power=p)
+    n_samples = 10
+    # We intentionally pick an arbitrary value for y_true and keep it fixed
+    # because the dropped constant term in the HalfTweedieLoss class depends on
+    # y_true.
+    y_true = np.full(shape=n_samples, fill_value=0.1)
+
+    # XXX: increasing the upper bound makes the test fail very easily, first
+    # with significantly different gradients, and then with different loss
+    # values.
+    raw_prediction = np.logspace(-10, -3, n_samples)
```

Comment:
I did not mean to use `np.logspace` but a `np.linspace` here. Otherwise taking `np.exp(raw_prediction)` would be very large. Fixed in 80bf4702223f624ffcd66625ed8f76cb966a8c08.